controllers/board-game-review-controllers.js

```diff
@@ -44,3 +44,16 @@ exports.getComments = (req, res, next) => {
         next(err);
     });
 };
+
+exports.newComment = (req, res, next) => {
```

Comment:
The name of this controller strays from your current convention - "newComment" sounds more like a variable name for a comment object - perhaps `postComment`?